README.md

```diff
@@ -1 +1,176 @@
-# toDo-review
\ No newline at end of file
+# Todo List
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Todo List</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Todo List- HTML, CSS, JS] <a name="about-project"></a>
+
+This is the first project of Module 2
+
```

Comment:
It would be better if you give  proper and more detailed description for the project.